tests/testthat/test-spam-dataloader.R

```diff
@@ -0,0 +1,16 @@
+if (requireNamespace("testthat", quietly = TRUE)) {
```

Comment:
You don't need this require statement here. `devtools::test()` will make sure testthat is laoded.